### DISCUSSION TITLE
How do you disable the "Assign to me" link for already assigned tasks?

### DISCUSSION THREAD

**Discussion by pilotspelman** [kanboard/kanboard#5490] [Q&A] [answered]
I would like to disable the "Assign to me" (quick)link for tasks already assigned to someone other than the logged-in user, provided the logged-in user has the role of Project Member. The logged-in user should be able to change the assignee by editing the task as usual, e.g., setting "Unassigned" for the task assigned to the logged-in user. This could also be an option under Custom roles.
Is there any "quick fix" I could try to accomplish this?

**Comment by creecros** [comment, marked as answer]
As far as I can tell, it already is set that way.
https://github.com/kanboard/kanboard/blob/616893832c11edfa72676953f35661deb1e9c3a4/app/Template/task/details.php#L88

**Reply by pilotspelman** [reply]
Thanks for pointing me in the right direction!
By changing line 88 to the code below, I got the "quick fix" I was asking for.
`<?php if (!($editable && $task['owner_id'] != 0)): ?>`
